CONFIGDOC.md

```diff
@@ -268,12 +276,13 @@ node {
 | testResultsSimulinkTest 	| Path to export Simulink Test Manager results in MLDATX format (requires Simulink Test license and is supported in MATLAB R2019a or later).<br/>**Example:** `'test-results/results.mldatx'` 	|
 | codeCoverageCobertura   	| Path to write code coverage report in Cobertura XML format.<br/>**Example:** `'code-coverage/coverage.xml'`     	|
 | modelCoverageCobertura  	| Path to write model coverage report in Cobertura XML format (requires Simulink Coverage™ license and is supported in MATLAB R2018b or later).<br/>**Example:** `'model-coverage/coverage.xml'`   	|
+| sourceFolder  	| Locations of the folders containing source code, relative to the project root folder. The specified folders and their subfolders are added to the top of the MATLAB search path. To generate a coverage report, MATLAB uses only the source code in the specified folders and their subfolders. <br/>**Example:** `'source'`<br/>**Example:** `["src/folderA", "src/folderB"]`   	|

```

Comment:
Is there a reason to use double quotes over single quotes for the source folder array example? I think either will work but single quotes would be more consistent with the rest of the document:

`['src/folderA', 'src/folderB']`

Reply:
Thanks for your comment. Changed it.

Reply:
Looks good.

Reply:
Actually it does , Jenkins has separate Pipeline type project like Matrix project other than freestyle
![image](https://user-images.githubusercontent.com/47204011/95330351-0f68b800-08c6-11eb-949d-33099011b5df.png)